### PR TITLE
Indented HTMl files

### LINKED ISSUE #97: Fix Indentation and clean up code.
### Expected results
Indented files
### Actual results
Some files not indented properly
### Steps to reproduce the issue
Update files
## Would you like to work on this
I am working to indent the HTMl files someone can go ahead and indent other thanks!

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Fixes #97
Fixed index.html and how-to-play.html
Proper indentation 
Moreover closed some un-closed tags

### CODE REVIEW

File: howtoplay.html [hunk 1]
```diff
@@ -1,54 +1,54 @@
-<!doctype html>
+  <!doctype html>
```
Comment (abishekvashok) [inline howtoplay.html]: Is it required we have double spaces at the top?

**Comment by niccokunzmann** [comment]
Thanks!

**Comment by niccokunzmann** [comment]
@abishekvashok I think, the small variances can be corrected now. If we merge fast, others can build on this.

**Comment by abishekvashok** [comment]
@niccokunzmann :D okay

**Pull request merged**